maistra/vendor/emsdk/emscripten_deps.bzl

```diff
@@ -2,6 +2,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@build_bazel_rules_nodejs//:index.bzl", "npm_install", "node_repositories")
 load(":revisions.bzl", "EMSCRIPTEN_TAGS")
 
+#load("//bazel:repositories.bzl", )
+load("@envoy//bazel:repositories.bzl", "envoy_dependencies", "BUILD_ALL_CONTENT")
```

Comment:
We should not touch files under the `vendor` directory. All these files come from Envoy and and changes made here will be overwritten when syncing with Envoy.